docs/applications/server.md

```diff
@@ -106,6 +106,10 @@ For more information on how to use our built in `cli` to interact with the serve
     - `message` - message to put into OP_RETURN commitment
     - `--hashMessage` - should the message be hashed before commitment
     - `--feerate <value>` - Fee rate in sats per virtual byte
+ - `lockunspent` `unlock` `transactions` - Temporarily lock (unlock=false) or unlock (unlock=true) specified transaction outputs.
+    - `unlock` - Whether to unlock (true) or lock (false) the specified transactions
+    - `transactions` - The transaction outputs and within each, the txid and vout
```

Comment:
Isn't it just clearer to say the transaction outpoints to unlock/lock? https://developer.bitcoin.org/reference/rpc/lockunspent.html#argument-2-transactions